docs/reference/esql/esql-rest.asciidoc

```diff
@@ -204,6 +204,30 @@ Which returns:
 }
 ----
 
+[discrete]
+[[esql-locale-param]]
+==== Returning localized results
+
+Use the `locale` parameter in the request body to return results (especially dates) formatted per the conventions of the locale.
+If `locale` is not specified, defaults to `en` (English).
```

Comment:
The default locale that ESQL uses is `en_US`.

Reply:
ISO-639 is just part of the solution - the JDK uses other ISOs as indicated [here](https://oracle.com/java/technologies/javase/jdk17-suported-locales.html). This also shows up in the default `en_US` (lang + country).

Reply:
In terms of usage, I actually couldn't get **locale** to work with codes of the form `xy_XY`, only `xy`.

Reply:
@astefan @costin 

The `locale` parameter works with parameters of format (case-insensitive)`xy` and `xy-XY`, but not `xy_XY`

Reply:
👍 you are correct, the parameter is a `LanguageTag` instance. And from https://www.oracle.com/java/technologies/javase/jdk17-suported-locales.html the Language Tag column shows `en-US`.

Reply:
I guess we can backport this to 8.12?